src/java.sql.rowset/share/classes/com/sun/rowset/RowSetResourceBundle_zh_TW.properties

```diff
@@ -157,7 +157,7 @@ xmlrch.errinsert = \u5EFA\u69CB\u63D2\u5165\u5217\u6642\u767C\u751F\u932F\u8AA4
 xmlrch.errinsdel = \u5EFA\u69CB insdel \u5217\u6642\u767C\u751F\u932F\u8AA4: {0}
 xmlrch.errupdate = \u5EFA\u69CB\u66F4\u65B0\u5217\u6642\u767C\u751F\u932F\u8AA4: {0}
 xmlrch.errupdrow = \u66F4\u65B0\u5217\u6642\u767C\u751F\u932F\u8AA4: {0}
-xmlrch.chars = \u5B57\u5143: 
+xmlrch.chars = \u5B57\u5143:\u0020
```

Comment:
Likely not needed, since the original and all other l10n versions of RowSetResourceBundle.properties do not have a trailing space for `xmlrch.chars`